exhibits/browserify/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "browserify-exhibit",
+  "version": "0.0.0",
+  "description": "A browserify/react-map-gl exhibit",
+  "scripts": {
+    "link": "(cd .. && rm -f node_modules && ln -s browserify/node_modules)",
```

Comment:
For what its worth, while this is clever, I think this should be removed. The purpose here is to make it super easy for someone to get started - either with browserify or with webpack. 

The simplest way to do that is to provide two complete example folders. Duplicating a file like app.js is not an issue.

This line is not easy to understand for a lot of people, and will not work/make sense when they try to copy this to start their own app.